bamboo/docker-compose-local.yml

```diff
@@ -7,13 +7,6 @@ services:
       - ./bootstrap-sftp.sh:/bootstrap-sftp.sh
       - ../packages/test-data/keys/ssh_client_rsa_key.pub:/etc/authorized_keys/user
       - ../packages/test-data:/data
-  pg_admin:
```

Comment:
I may re-add this to a development compose at some point soon.     Adding comment so I can make slackbot remind me (we're living in the future)